docs/index.md

```diff
@@ -37,6 +37,7 @@ execution environment and reporting tool.
 [libFuzzer]: https://llvm.org/docs/LibFuzzer.html
 [AFL++]: https://github.com/AFLplusplus/AFLplusplus
 [Honggfuzz]: https://github.com/google/honggfuzz
+[Centipede]: https://github.com/google/centipede
```

Comment:
It doesn't make sense to define what `[Centipede]` points to if you haven't put `[Centipede]` anywhere on this page.